### PR TITLE
Add help command

### PR DESCRIPTION
**Pull request by ghost**
**-- Changes --**
- Added /help command.
- Made chat room description show more useful info like the Discord server invite.

### CODE REVIEW

**Comment by ghost** [comment]
Does alert popup allow HTML tags? Also, is there a function to send a system chat message to yourself? It would be better to use that instead of a popup on help command.

File: Server/Game/Commands/Misc/AlertCommand.cs [hunk 1]
```diff
@@ -14,21 +14,28 @@ internal class AlertCommand : ICommand
 
         public void OnCommand(ICommandExecutor executor, string label, ReadOnlySpan<string> args)
         {
-            if (args.Length >= 2)
+            if (executor is ClientSession session)
```
Comment (aromaa) [inline Server/Game/Commands/Misc/AlertCommand.cs]: This makes no sense, the console should be able to send alerts to other players.

File: Server/Game/Commands/Misc/HelpCommand.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+namespace Platform_Racing_3_Server.Game.Commands.Misc
+{
+    internal class HelpCommand : ICommand
+    {
+        public string Permission => null;
+
+        public void OnCommand(ICommandExecutor executor, string label, ReadOnlySpan<string> args)
+        {
+            if (executor is ClientSession session)
```
Comment (aromaa) [inline Server/Game/Commands/Misc/HelpCommand.cs]: This should also be available on console

File: Server/Game/Commands/Misc/HelpCommand.cs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            {
+                executor.SendMessage("<b><u>Developer / Administrator / Manager</u></b>"
```
Comment (aromaa) [inline Server/Game/Commands/Misc/HelpCommand.cs]: This is pretty terrible way to implement this, going through all available commands and checking for HasPermission is the proper way. Also adding Description getter to ICommand is better way to implement this.

**Review by aromaa — CHANGES_REQUESTED** [review]

**Comment by aromaa** [comment]
> Does alert popup allow HTML tags? Also, is there a function to send a system chat message to yourself? It would be better to use that instead of a popup on help command.

The alert popup supports HTML tags. If you want to send single session personalized message there is the ChatOutgoingMessage packet. Tho that probs should be implemented to SendMessage that checks if the user is in chat room and if not instead use alert.

File: Server/Game/Commands/Misc/AlertCommand.cs (hunk 1 above)
Comment (ghost) [reply Server/Game/Commands/Misc/AlertCommand.cs]: Whoopsies, my bad. Thanks! :+1:

File: Server/Game/Commands/Misc/HelpCommand.cs (hunk 2 above)
Comment (ghost) [reply Server/Game/Commands/Misc/HelpCommand.cs]: :+1:

File: Server/Game/Commands/Misc/HelpCommand.cs (hunk 3 above)
Comment (ghost) [reply Server/Game/Commands/Misc/HelpCommand.cs]: Yup, that's what I was talking about at the top. Just made it this way temporarily until I make the proper one. I'll let you know once it's done. :smiley:

**Pull request closed without merging**